### DISCUSSION TITLE
Show that a post has been updated

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Revisions#917] [Feature Requests]
One user requested the option to show that a post has been updated.

They originally suggested PublishPress:

> We've created a custom status called "Updated", the purpose of which is to show that a post has not only been published, but updated since publication
> 
> the issue we're having is that it seems none of the PP post statuses are public, so it's effectively a draft and only viewable from the dashboard

My thinking is that this would be best in Revisionary. This could be similar to https://github.com/publishpress/Revisionary/issues/15

Maybe there could be a widget or shortcode showing if a post has been updated, together with some optional details

**Comment by JoshJourney** [comment]
The widget title could be called "Statuses" but start out as simply displaying "Updated". Eventually it could include "Pending Revisions: 3", "Popular", and anything relevant to Revisionary.
